package.json

```diff
@@ -31,5 +31,10 @@
     "babel-cli": "~6.14.0",
     "babel-plugin-add-module-exports": "~0.2.1",
     "babel-preset-es2015": "~6.14.0"
+  },
+  "jspm": {
+    "main": "src/index.js",
```

Comment:
SystemJS transpiles (where needed) in the browser or JSPM does this beforehand when building or bundling. So there is no reason to use pre-transpiled code with JSPM/SystemJS.
With this change I can much more easily work on this library. I can easily fork your library, clone it, then link to that local copy from my app (this creates a symlink to my cloned local copy). I can then edit that copy (in ES6) and directly see the effects in my app. 
This is how I created and tested the last two pull requests. 

I checked if it still worked when installing it (instead of linking it) by installing the develop branch of [our fork](https://github.com/Doodle3D/superlogin-client) on github.